conf/modules/annotate_cadd.config

```diff
@@ -0,0 +1,43 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
+        ext.when            = Conditional clause
+----------------------------------------------------------------------------------------
+*/
+
+//
+// CADD annotation
+//
+
+process {
+
+    withName: '.*:ANNOTATE_CADD:.*' {
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: '.*:ANNOTATE_CADD:BCFTOOLS_VIEW' {
+        ext.args   = { "--output-type z --types indels,other" }
```

Comment:
Just out of curiosity, how big are the indels in the SNV/INDEL vcf coming from ONT data? Can they be several 10s of kb and would that severely affect how long it takes to run CADD?

Reply:
I haven't tested this yet, will do before merging.